src/generic/datavgen.cpp

```diff
@@ -51,6 +51,8 @@
 #include "wx/selstore.h"
 #include "wx/stopwatch.h"
 #include "wx/weakref.h"
+#include "wx/hashmap.h"
+#include "wx/dynarray.h"
```

Comment:
This really shouldn't be used in the new code, just use `wx/vector.h` and `wxVector`, which is compatible with `std::vector`, instead